src/util.py

```diff
@@ -1,10 +1,24 @@
 import yaml
+import argparse
 
 def load_hparams(path):
     with open(path, 'r') as stream:
         try:
-            defaults = yaml.safe_load(stream)['hparams']
+            hparams = yaml.safe_load(stream)['hparams']
         except yaml.YAMLError as e:
             print(e)
 
-    return defaults
+    return hparams
```

Comment:
This is not how to handle errors: the `['hparams']` might throw a KeyError, and if the parsing gives a YAMLError, the return will give a NameError.